tests/test_SklearnNaiveBayesConverter.py

```diff
@@ -49,8 +49,7 @@ def test_model_multinomial_nb_multiclass(self):
         dump_data_and_model(X, model, model_onnx, basename="SklearnMclMultinomialNB-OneOff",
                             allow_failure="StrictVersion(onnxruntime.__version__) <= StrictVersion('0.1.3')")
 
-    #@unittest.skipIf(StrictVersion(onnx.__version__) <= StrictVersion('1.3'), 'Need Greater Opset 9')
-    @unittest.skip("Doesn't work")
+    @unittest.skip(reason="Type 'tensor(float)' of input parameter (input) of operator (ConstantOfShape) in node (ConstantOfShape) is invalid.")
```

Comment:
Is it possible to split it over multiple lines, it requires scrolling to see the entire line.